File: code.leetcode/shuzu/easy/e283.移动零.py
class Solution(object):
    def moveZeroes(self, nums):
                #先删掉再补全
            #双指针
        left=0
        i=0
        while i<len(nums):
            if nums[i]!=0:
                nums[left]=nums[i]
                left+=1
            i+=1
            

        #补全
        j=left
        while j<len(nums):
            nums[j]=0
            j+=1

File: code.leetcode/shuzu/easy/e283.移动零.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
//先删掉再补全
	//双指针
int left=0;
for (int i=0;i<nums.size();++i){
    if (nums[i]!=0){
        nums[left]=nums[i];
        ++left;
    }
}
//补全
for (int j=left;j<nums.size();++j){
    nums[j]=0;
}
    }
};